source/locales/de.json

```diff
@@ -133,7 +133,7 @@
   "Email Encryption": "E-Mail-Verschlüsselung",
-  "email-encryption-notes": "<p>&ldquo;Pretty Good Privacy (PGP, deutsch sinngemäß „Ziemlich gute Privatsphäre“) ist ein Programm zur Verschlüsselung und zum Unterschreiben von Daten.&rdquo;</p><p>&mdash; <a href=\"https://de.wikipedia.org/wiki/Pretty_Good_Privacy\">Wikipedia</a><br><p>PGP wird häufig zum signieren, verschlüsseln und entschlüsseln von Texten, E-Mails, Dateien, Ordnern und ganzen Festplattenpartitionen genutzt, um die Sicherheit von E-Mailkommunikation zu erhöhen.</p><footer>&mdash; <cite><a href='https://de.wikipedia.org/wiki/Pretty_Good_Privacy'>Wikipedia</a></cite></footer></blockquote><p>PRISM Break empfiehlt die S/MIME E-Mail Verschlüsselung nicht, da es Drittanbieter-Zertifikate von zentralen Autoritäten benötigt. Mehr <a href='https://www.enigmail.net/forum/viewtopic.php?f=7&amp;t=67'>hier (Englisch)</a>.</p><p><strong>APG</strong> wurde in den letzten drei Jahren nicht aktualisiert. Experimentelle PGP-Projekte, die es wert sind, anzusehen, sind <a href='https://github.com/dschuermann/openpgp-keychain'>OpenPGP Keychain</a> (ein APG-Zweig) und <a href='https://dev.guardianproject.info/projects/gpgandroid/wiki'>GnuPG für Android</a> von dem Guardian Project.</p><p>Read the <a href='https://emailselfdefense.fsf.org/'>Email Self-Defense</a> guide by the Free Software Foundation to learn how to encrypt your email messages.</p>",
+  "email-encryption-notes": "<p>&ldquo;Pretty Good Privacy (PGP, deutsch sinngemäß „Ziemlich gute Privatsphäre“) ist ein Programm zur Verschlüsselung und zum Unterschreiben von Daten.&rdquo;</p><p>&mdash; <a href=\"https://de.wikipedia.org/wiki/Pretty_Good_Privacy\">Wikipedia</a><br><p>PGP wird häufig zum signieren, verschlüsseln und entschlüsseln von Texten, E-Mails, Dateien, Ordnern und ganzen Festplattenpartitionen genutzt, um die Sicherheit von E-Mailkommunikation zu erhöhen.</p><footer>&mdash; <cite><a href='https://de.wikipedia.org/wiki/Pretty_Good_Privacy'>Wikipedia</a></cite></footer></blockquote><p>PRISM Break empfiehlt die S/MIME E-Mail Verschlüsselung nicht, da es Drittanbieter-Zertifikate von zentralen Autoritäten benötigt. Mehr <a href='https://www.enigmail.net/forum/viewtopic.php?f=7&amp;t=67'>hier (Englisch)</a>.</p><p><strong>OpenKeychain</strong> zusammen mit <strong>K-9 Mail</strong> bietet Ende-zu-Ende-Verschlüsselung für E-Mails. Ein experimentelles Projekt ist <a href='https://dev.guardianproject.info/projects/gpgandroid/wiki'>GnuPG für Android</a> von dem Guardian Project.</p><p>Read the <a href='https://emailselfdefense.fsf.org/'>Email Self-Defense</a> guide by the Free Software Foundation to learn how to encrypt your email messages.</p>",
```

Comment:
@dschuermann ummm, not sure. I think probably the best course of action is to remove the outdated translations (which will cause them to fallback to English). /cc @nylira